src/assets/styles/styles.11ty.js

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright the Fluidic Eleventy copyright holders.
+
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/fluidic-11ty/raw/master/AUTHORS.md.
+
+Licensed under the New BSD license. You may not use this file except in compliance with this License.
+
+You may obtain a copy of the New BSD License at
+https://github.com/fluid-project/fluidic-11ty/raw/master/LICENSE.md.
+*/
+"use strict";
+
+const fs = require("fs");
+const fg = require("fast-glob");
+const path = require("path");
+const sass = require("sass");
+const stylus = require("stylus");
+const autoprefixer = require("autoprefixer");
+const postcss = require("postcss");
+const CleanCSS = require("clean-css");
+const cssesc = require("cssesc");
+
+const isProd = process.env.ELEVENTY_ENV === "production";
+
+/**
+ * @see https://github.com/maxboeck/eleventastic/blob/master/src/assets/styles/styles.11ty.js
+ */
+
+module.exports = class {
+    // Configure the transform.
+    async data() {
+        const entryPaths = await fg("./src/assets/styles/style.{css,scss,styl}");
+        const entryPath = entryPaths[0];
+        return {
+            permalink: '/assets/styles/style.css',
+            eleventyExcludeFromCollections: true,
+            entryPath
+        };
+    }
+
+    // Compile SCSS to CSS.
+    async compileScss(options) {
+        return new Promise((resolve, reject) => {
+            const config = {
+                file: options.entryPath,
+                sourceMap: !isProd ? 'out.map' : false,
+                sourceMapEmbed: !isProd || false,
+                outputStyle: isProd ? 'compressed' : 'expanded'
+            };
+
+            sass.render(config, (err, result) => {
+                if (err) {
+                    reject(err);
+                } else {
+                    const processedCss = this.processCss({str: result.css, entryPath: options.entryPath});
+                    resolve(processedCss);
+                }
+            });
+        });
+    }
+
+    // Compile Stylus to CSS.
+    async compileStylus(options) {
+        return new Promise((resolve, reject) => {
+            const sourceMap = isProd ? false : { inline: true };
```

Comment:
same comments as above about source maps.

Reply:
Same issue— because it's inlined it does affect performance.

Reply:
Instead of inline can we have a separate file generated?

Reply:
It's not possible with the method I'm using, which writes the generated CSS to a single output file using Eleventy. If separate map files are required, that would require a different approach entirely.

Reply:
An *.11ty.js file maps to a single output file. See: https://www.11ty.dev/docs/languages/javascript/

That output path is defined in the `data()` method here: https://github.com/fluid-project/fluidic-11ty/pull/30/files/8b004c5046de41a0c7821439c3525ddc75f624b5#diff-f94be944bbcca875207aad9633811196abeafd7c3b838a505b6db1bdd81c1850R36

If we wanted to generate multiple output files, we couldn't use Eleventy to run a preprocessor but would have to rework this asset pipeline to use some other method (probably Webpack as is used for JavaScript in #31).